catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -554,11 +555,14 @@ public Response getDocument(@Encoded @PathParam("sourceid") String encodedSource
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.BAD_REQUEST);
                 // The catalog framework will throw this if any of the transformers blow up. We need to
-
                 // catch this exception
                 // here or else execution will return to CXF and we'll lose this message and end up with
                 // a huge stack trace
                 // in a GUI or whatever else is connected to this endpoint
+            } catch (DataUsageLimitExceededException e) {
+                String exceptionMessage = "Unable to process request. Data usage limit exceeded: ";
```

Comment:
From the test instructions, it seems to indicate that the "usage limit" could be changed. Would it be appropriate to provide a recommendation to the user that they could modify this setting?